selfdrive/car/hyundai/fingerprints.py

```diff
@@ -1003,6 +1003,7 @@
   CAR.HYUNDAI_TUCSON_4TH_GEN: {
     (Ecu.fwdCamera, 0x7c4, None): [
       b'\xf1\x00NX4 FR_CMR AT CAN LHD 1.00 1.01 99211-N9100 14A',
+      b'\xf1\x00NX4 FR_CMR AT CAN LHD 1.00 1.00 99211-N9260 14Y',
```

Comment:
Once the 2025 Tucson comes out in June we can know what Hyundai does when they need to wrap from 14Z and then we can enable fuzzy fingerprinting for this car to prevent the need for these PRs!